includes/blocks/class-sensei-course-outline-block.php

```diff
@@ -185,6 +190,40 @@ private function get_module_block_html( $block ) {
 		';
 	}
 
+	/**
+	 * Get progress indicator HTML.
+	 *
+	 * @param array $module_id The module id.
+	 * @param int   $course_id The course id.
+	 *
+	 * @return string Module HTML
+	 */
+	private function get_progress_indicator( $module_id, $course_id ) {
+
+		$module_progress = Sensei()->modules->get_user_module_progress( $module_id, $course_id, get_current_user_id() );
+
+		if ( empty( $module_progress ) ) {
+			return '';
+		}
+
+		if ( $module_progress < 100 ) {
+			$module_status   = __( 'COMPLETED', 'sensei-lms' );
+			$indicator_color = '#63a95f';
```

Comment:
The color is inline to more easily support setting it with block attributes in the future. I could add a CSS class instead.